ipt_connect/IPTdev/parameters.py

```diff
@@ -1,7 +1,16 @@
 # -*- coding: utf-8 -*-
 
 # Views parameters
-app_version = "dev"     # keyword for url parsing
+app_version = "IPTdev"[3:]     # keyword for url parsing
```

Comment:
That's indeed a dirty hack.

Reply:
It's waiting for a hero to be fixed. Looks like a good issue for a newbie (much work, few difficulties). See #77.